templates/zerver/emails/followup_day1.txt

```diff
@@ -1,33 +1,43 @@
-Hi there,
+{{ _('Hi there,') }}
 
-Welcome to Zulip! A few tips to get you started:
+{{ _('Welcome to Zulip! A few tips to get you started:') }}
 
+{% trans %}
 * Zulip works best when it's always open, so we suggest pinning a browser
-tab or downloading our desktop and mobile apps (https://zulipchat.com/apps).
+  tab or downloading our desktop and mobile apps (https://zulipchat.com/apps).
+{% endtrans %}
 
+{% trans %}
 You'll need an Organization URL when installing the apps. That's
 {{ realm_uri }}
+{% endtrans %}
```

Comment:
We should move the `realm_uri` out of the translated part; that way this string will be the same as the HTML one.